src/js/api/core-shim.js

```diff
@@ -85,9 +85,9 @@ Object.assign(CoreShim.prototype, {
         model.setProvider = function() {};
 
         // Create/get click-to-play media element, and call .load() to unblock user-gesture to play requirement
-        const mediaElement =
-            model.attributes.mediaElement = getMediaElement(this.originalContainer);
-        mediaElement.load();
+
+        const mediaPool = MediaElementPool();
+        mediaPool.prime();
```

Comment:
This isn't a singleton. Every time `MediaElementPool()` is called, a new set of three tags is returned.